### PR TITLE
Refresh to make it work with 2016 gems

### PR DESCRIPTION
**Pull request by tomash**
Great gem, exactly the set of abstractions for Repository patterns that I looked for!
We're going to use it in one of our projects here at Rebased, so I've forked the repo.

You can consider the project no-longer-discontinued, of course if you'd like to merge PRs and publish new gem versions :smile: 

Should I bump the version too?

**Actual PR:**

Refresh to make it work with Ruby 2.2+ and current versions of Mongoid. Namely:

* NoMethodError message matched with regexp due to changed format
* ActiveRecord and ActiveModel dependencies consistently bumped to >= 4.1.4
* .rspec for colored output by default

### CODE REVIEW

**Comment by efreesen** [comment]
Thank you, it's great to hear that my work has helped you guys.

Sure, I can merge your PR and publish a new gem version, no problem.

Let me know if you need anything else or if I can merge it.

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
-    - rvm: jruby-19mode
-
+  - 2.2.2
+  - 2.3.0
+#  - jruby-9.1.5.0
+#before_script:
+#  - jdk_switcher use oraclejdk8
+before_install:
+  - gem install bundler
+  - gem install bundler
```
Comment (efreesen) [inline .travis.yml]: Is this line duplicated?

File: .rspec [hunk 2]
```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```
Comment (efreesen) [inline .rspec]: Maybe add a newline to the end of the file?

Comment (tomash) [inline .rspec]: good point, will do

**Pull request merged**